src/components/DevicesView.jsx

```diff
@@ -10,14 +10,14 @@ import semver from 'semver'
 import ActionMenu, {
   ActionMenuHeader,
   ActionMenuItem
-} from 'cozy-ui/transpiled/react/ActionMenu'
-import Alerter from 'cozy-ui/transpiled/react/Alerter'
+} from 'cozy-ui/transpiled/react/deprecated/ActionMenu'
+import Alerter from 'cozy-ui/transpiled/react/deprecated/Alerter'
 import IconButton from 'cozy-ui/transpiled/react/IconButton'
 import DotsIcon from 'cozy-ui/transpiled/react/Icons/Dots'
 import GearIcon from 'cozy-ui/transpiled/react/Icons/Gear'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
-import MuiButton from 'cozy-ui/transpiled/react/MuiCozyTheme/Buttons'
+import MuiButton from 'cozy-ui/transpiled/react/deprecated/Button'
```

Comment:
Warning here it's not the good button

it should be 

`import MuiButton from 'cozy-ui/transpiled/react/Button'`

See https://github.com/cozy/cozy-libs/pull/2218#discussion_r1247697161

@Merkur39 you said you also had this issue last time because you ran the codemod is the wrong order ?

Reply:
Exactly, it's very important to run cozy-ui codemods v87 then v88 !

Reply:
slim I'm redoing the codemods in the right order